osu.Game.Rulesets.Osu/Difficulty/Skills/Speed.cs

```diff
@@ -35,7 +36,7 @@ public Speed(Mod[] mods, double hitWindowGreat)
 
         protected override double StrainValueAt(DifficultyHitObject current)
         {
-            currentStrain *= strainDecay(current.DeltaTime);
+            currentStrain *= strainDecay(((OsuDifficultyHitObject)current).StrainTime);
```

Comment:
Wondering whether we should do the same here for every skill 🤔

Reply:
Probably doesn't make much of a difference for other skills, but if the other skills calculate strain based on `StrainTime` and not `DeltaTime` then it would make more sense to decay with `StrainTime` as well.